src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -185,6 +184,30 @@ describe("Editor Page Component", () => {
         expect(saveProjectSpy).toBeCalledWith(expect.objectContaining(partialProject));
     });
 
+    describe("Editing Video Assets", () => {
+        it("Child assets are not included within editor page state", async () => {
+            const testProject = MockFactory.createTestProject("TestProject");
+            const videoAsset = MockFactory.createVideoTestAsset("TestVideo");
+            const videoFrames = MockFactory.createChildVideoAssets(videoAsset);
+            const projectAssets = [videoAsset].concat(videoFrames);
+            testProject.assets = _.keyBy(projectAssets, (asset) => asset.id);
+
+            const store = createStore(testProject, true);
+            const props = MockFactory.editorPageProps(testProject.id);
+
+            const wrapper = createComponent(store, props);
+            const editorPage = wrapper.find(EditorPage).childAt(0) as ReactWrapper<IEditorPageProps, IEditorPageState>;
+
+            await MockFactory.flushUi();
+
+            expect(editorPage.state().assets.length).toEqual(testAssets.length + 1);
+            expect(editorPage.state().selectedAsset.asset).toEqual({
```

Comment:
`selectedAsset` should be rename (it was confusing to see what I thought is an asset object containing an asset property)

Reply:
The `selectedAsset` on the state is the selected asset that is currently loaded in the editor.